thirdparty/vars.sh

```diff
@@ -283,10 +283,10 @@ CCTZ_SOURCE="cctz-2.3"
 CCTZ_MD5SUM="209348e50b24dbbdec6d961059c2fc92"
 
 # datatables, bootstrap 3 and jQuery 3
-DATATABLES_DOWNLOAD="https://datatables.net/download/builder?bs-3.3.7/jq-3.3.1/dt-1.10.22"
+DATATABLES_DOWNLOAD="https://datatables.net/download/builder?bs-3.3.7/jq-3.3.1/dt-1.10.23"
```

Comment:
Is there a place to store historical versions of DATATABLES instead of following the official website to upgrade to the latest when the compilation fails every time.

Reply:
Sorry, I do not found the place for storing historical versions now.